patches/chrome-tools-build-mac-BUILD.gn.patch

```diff
@@ -1,12 +1,16 @@
 diff --git a/chrome/tools/build/mac/BUILD.gn b/chrome/tools/build/mac/BUILD.gn
-index cd080aee2da3ecdb86738a457d31c70305c83150..f3139fe1f223fdffa0f8a1183f86178988d4ad3b 100644
+index cd080aee2da3ecdb86738a457d31c70305c83150..bd6819cdd396e92f7622335a7f03514bc426fe12 100644
 --- a/chrome/tools/build/mac/BUILD.gn
 +++ b/chrome/tools/build/mac/BUILD.gn
-@@ -4,6 +4,7 @@
+@@ -4,6 +4,11 @@
  
  executable("infoplist_strings_util") {
    configs += [ "//build/config/compiler:wexit_time_destructors" ]
 +  configs += [ "//brave/common:constants_configs" ]
++
++  # Avoid a build error when cross-compiling from Linux to macOS:
```

Comment:
Your suggested changes work for me. However: What's the advantage of making it a single-line patch? The patch "region" is already contiguous, even tough it's multiple lines. So there shouldn't be any more conflicts than if it were on a single line, no?

Reply:
We always try to do single-line patches, it's easier to maintain. @bridiver can share some details why we prefer it this way.

Reply:
Thank you. I found "ignore lint line length rules to squash patches onto one line whenever possible" in the Wiki [here](https://github.com/brave/brave-browser/wiki/Patching-Chromium#patch-the-chromium-files). But it doesn't give an explanation as to why. (Maybe it's because then the line numbers in the patch file stay relatively constant?) Would love to hear about the motivation for single-line patches from @bridiver, which I'd then add to the Wiki. In the meantime, I'll just do it this way.

Reply:
Resolved in 17d0938f0a8d447bf6ff61fa1201edcdeff78115.